pypeit/pypmsgs.py

```diff
@@ -23,7 +23,7 @@
 #pypeit_logger = None
 
```

Comment:
This sets the verbosity for the people in the list.  I removed myself because I want to see what normal users see.  In terms of the larger point, we might consider adding some stuff to our main `__init__.py` file that credits developers, etc.  See e.g.: https://stackoverflow.com/questions/1523427/what-is-the-common-header-format-of-python-files